ch-applications/satisfiability-grover.ipynb

```diff
@@ -237,7 +237,7 @@
    "source": [
     "# Load our saved IBMQ accounts and get the ibmq_16_melbourne backend\n",
     "from qiskit import IBMQ\n",
-    "IBMQ.load_accounts()\n",
+    "IBMQ.load_account()\n",
     "\n",
     "backend = IBMQ.get_backend('ibmq_16_melbourne')"
```

Comment:
Thanks for the changes @MozammilQ 

Can you update this line to `backend = IBMQ.get_provider().get_backend('ibmq_16_melbourne')`? Otherwise, the update above is not sufficient.

Reply:
My pull request:   https://github.com/Qiskit/qiskit-textbook/pull/28
has already been mergerd. So, first of all THANK YOU ! I feel very happy about this.


The above pull request addressed the problem stated by you here:
IBMQ.get_provider().get_backend('ibmq_16_melbourne')


This pull request will only set the this:  IBMQ.load_accounts() -->  IBMQ.load_account()
So, I humbly requsest you to merge this as well.

Now, I know, I should have done both the changes in one pull request only ;)

Reply:
This has already been solved by another PR -- the file now has the correct credential and backend retrieving procedure.